locales/en/common.json

```diff
@@ -1,3 +1,67 @@
 {
-  "back": "Back"
+  "actions": "Actions",
+  "active": "Active",
+  "all_your_apps_using_this_connection_will_stop_working": "All your apps using this connection will stop working.",
+  "back": "Back",
+  "cancel": "Cancel",
+  "choose_an_identity_provider": "Choose an Identity Provider",
+  "clear_events": "Clear Events",
+  "client_error": "client error",
+  "close_sidebar": "Close sidebar",
+  "confirmation_modal_description": "This action cannot be undone. This will permanently delete the Connection.",
+  "create_directory": "Create Directory",
+  "create_new": "Create New",
+  "create_sso_connection": "Create SSO Connection",
+  "delete": "Delete",
+  "delete_the_connection": "Delete the Connection?",
+  "delete_this_connection": "Delete this connection",
+  "directory_id": "Directory ID",
+  "directory_name": "Directory name",
+  "directory_provider": "Directory provider",
+  "directory_sync": "Directory Sync",
+  "edit_sso_connection": "Edit SSO Connection",
+  "email": "Email",
+  "enable_webhook_events_logging": "Enable Webhook events logging",
+  "enterprise_readiness_for_b2b_saas_straight_out_of_the_box.": "Enterprise readiness for B2B SaaS, straight out of the box.",
```

Comment:
Please remove the period from the end of the string `enterprise_readiness_for_b2b_saas_straight_out_of_the_box `

Reply:
done